sdk/python/the0/state.py

```
@@ -0,0 +1,182 @@
+    _validate_key(key)
+    filepath = _get_key_path(key)
+    try:
+        with open(filepath, "r") as f:
+            return json.load(f)
+    except FileNotFoundError:
+        return default
+    except json.JSONDecodeError:
+        return default
+
+
+def set(key: str, value: Any) -> None:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)